XamlControlsGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -37,7 +34,21 @@
             <local:ControlExample.Substitutions>
                 <local:ControlExampleSubstitution Key="IsActive" Value="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" />
             </local:ControlExample.Substitutions>
-
+        </local:ControlExample>
+        <local:ControlExample HeaderText="A determinate progress ring.">
+            <StackPanel x:Name="Control2" Orientation="Horizontal">
+                <muxc:ProgressRing Width="60" x:Name="ProgressRing2" IsIndeterminate="False"/>
+                <TextBlock x:Name="Control2Output" Style="{ThemeResource OutputTextBlockStyle}" Width="60" TextAlignment="Center" />
```

Comment:
Is this TextBlock actually used? I'm not seeing any control output...

Reply:
It's separating the NumberBox from the ring.

Reply:
In that case, you just want a margin. Delete the TextBlock and update the ProgressRing to something like

```
                <muxc:ProgressRing Width="60" x:Name="ProgressRing2" IsIndeterminate="False" Margin="0,0,60,0"/>
```